public/javascripts/Gallery/css/cards.css

```diff
@@ -1,75 +1,182 @@
 .gallery-card {
     position: relative;
     display: inline-block;
-    border-style: solid;
-    border-width: 1px;
-    padding: 5px;
-    border-radius: 5px;
-    margin: 2px;
-    width: 360px;
+    padding: 0px;
+    margin: 5px;
+    border-radius: 20px;
+    width: 22vw;
     height: auto;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.1), 0 6px 20px 0 rgba(0, 0, 0, 0.05);
+    grid-area: 'gallery-card';
```

Comment:
Browser and IDE are both complaining that `'gallery-card'` is an invalid property value for `grid-area`. Is that right or do you know what this is doing?

Reply:
This was based on an older version of CSS-Grid, but it is now ineffective, so i'll remove it

Reply:
okay this one is still an issue in a lot of places and I'm not 100% what's going on with it so I don't want to just touch it without thinking

Reply:
This seems to be resolved, so I'm gonna close this.